django_mongoengine/mongo_admin/views.py

```diff
@@ -124,7 +124,7 @@ def get_ordering(self, request=None, queryset=None):
 
     def _lookup_param_1_3(self):
         lookup_params = self.params.copy() # a dictionary of the query string
-        for i in (ALL_VAR, ORDER_VAR, ORDER_TYPE_VAR, SEARCH_VAR,
```

Comment:
This change looking suspicious. This is query for django-1.3, and i think it's better to just remove it, as we don't support 1.3, minimum version is 2.2.

Reply:
right, removed the own method. And I found the method get_filters return 5 values dict in Django 4, so I also changed the code. https://github.com/django/django/blob/aff79be03a8c787ba0cc7e00dcec9eeb8ad01c87/django/contrib/admin/views/main.py#L452